utils/stress_test/stress_test.py

```diff
@@ -0,0 +1,225 @@
+    else:
+        request_data = request.data
+    g_queue.put(request_data, block=True)
+    return "", 200
+
+
+def request_processor_loop(queue: mp.Queue, test_name: str, num_files, main_pid: int):
+    # request data processing loop
+    test_sequences = {}
+    unrecognized_lines = 0
+    start_ts = timer()
+    last_report_ts = timer()
+    log = logging.getLogger(test_name)
+    while True:
+        if last_report_ts + 5 < timer():
+            last_report_ts = timer()
+            if check_test_state(
+                test_sequences, num_files, start_ts, unrecognized_lines
+            ):
+                log.info(f"FINISHED in {timer() - start_ts:.0f} sec")
+                os.kill(main_pid, signal.SIGINT)
+        try:
+            request_data = queue.get(block=True, timeout=1)
+        except Empty:
+            continue
+        data_str = None
+        try:
+            data_str = request_data.decode("utf-8")
+            data_json = json.loads(data_str)
+            lines = data_json["lines"]
+        except Exception as e:
+            log.error(repr(e))
+            log.error(f"line: '{data_str}'")
+            continue
+        for l in lines:
+            try:
+                line_obj = json.loads(l["line"])
+                seq_name: str = line_obj.get("seq_name")
+                if seq_name and seq_name.startswith(test_name):
+                    test_seq_obj = test_sequences.get(seq_name)
+                    if not test_seq_obj:
+                        test_seq_obj = TestSeq(seq_name, line_obj["total_lines"])
+                        test_sequences[seq_name] = test_seq_obj
+                    test_seq_obj.line_ids.append(line_obj["line_id"])
+                else:
+                    unrecognized_lines = unrecognized_lines + 1
+            except Exception as e:
+                log.error(repr(e))
+                pass
+
+
+def check_test_state(
+    test_sequences: typing.Dict[str, TestSeq],
+    num_files: int,
+    start_ts: float,
+    unrecognized_lines: int,
+) -> bool:
+    num_total_seq = num_files
+    num_received_seq = 0
+    num_completed_seq = 0
+    num_total_lines = 0
+    num_received_lines = 0
+    num_duplicate_lines = 0
+    run_time = timer() - start_ts
+    for seq in test_sequences.values():
+        num_received_seq = num_received_seq + 1
+        num_total_lines = num_total_lines + seq.total_lines
+        num_received_lines = num_received_lines + len(seq.line_ids)
+        if len(seq.line_ids) >= seq.total_lines:
+            if (
+                len(set(seq.line_ids)) == seq.total_lines
+                and len(seq.line_ids) == seq.total_lines
+            ):
+                num_completed_seq = num_completed_seq + 1
+                continue
+            if len(set(seq.line_ids)) <= seq.total_lines:
+                num_duplicate_lines = (
+                    num_duplicate_lines + seq.total_lines - len(set(seq.line_ids))
+                )
+    line_rate = num_total_lines / run_time
```

Comment:
line_rate should be `num_received_lines / run_time`
```suggestion
    line_rate = num_received_lines / run_time
```